core/domain/user_services.py

```diff
@@ -2169,11 +2168,6 @@ def create_login_url(return_url):
     Returns:
         str. The correct login URL that includes the page to redirect to.
     """
-    # TODO(#11462): Delete this function. Pre-#11462, we needed this because we
```

Comment:
Shouldn't this entire function be deleted, per the TODO?

Reply:
Ah, I actually think it's worth keeping since our backend code needs to redirect users to the login page anyway. Because of this, I think it's cleaner to keep it around.